django/forms/fields.py

```diff
@@ -575,6 +577,8 @@ def clean(self, data, initial=None):
             data = None
         if not data and initial:
             return initial
+        if self.multiple:
+            return [super(FileField, self).clean(value) for value in data]
```

Comment:
Why not just `super().clean(value)`?

Reply:
You can't call super inside a generator without a specific target and instance. I will make sure to add a comment.